### DISCUSSION TITLE
Cannot get the right stitching panorama result

### DISCUSSION THREAD

**Discussion by MedIWA** [OpenStitching/stitching#169] [General]

**Comment by lukasalexanderweber** [comment]
Do you have sample images or can explain your problem in more detail. Have you run the stitching in verbose mode?

**Comment by MedIWA** [comment]


**Reply by lukasalexanderweber** [reply]
I think the stitching won't work with the black borders without image data. You should cut out the the image data beforehand

**Comment by MedIWA** [comment]


**Reply by lukasalexanderweber** [reply]
This is just one image, send another one that matches to this one so I can help

**Reply by lukasalexanderweber** [reply]
those images are not suitable for stitching
